hw1-app/src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+class Button extends React.Component {
+    render() {
+        return (
+            <button className="button" onClick={()=>
+                this.props.sign === "+" ? 
+                this.props.handler(1)
+                : 
+                this.props.handler(-1)
+            }>{this.props.sign}</button>
+        );
+    }
+}
+
+export default Button
```

Comment:
For this component and all others where you use this.props.something.
Please investigate destructuring  concept in ES6.
Example: 
`const { handler, sign } = this.prop;`